### ISSUE TITLE
Quest: The Princess Trapped

### ISSUE THREAD

**Issue by underlord-oberon** [plantain-gnocchis/wow-voiceover-tbc-wotlk#62]
Add-on version: AI_VoiceOver_v1.4.5
Data version: AI_VoiceOverData_VanillaExtra_v1.4

Arathi Highlands / Object: Shards of Myzrael / Quest: The Princess Trapped
Link: https://www.wowhead.com/wotlk/quest=642/the-princess-trapped

Description: Despite the voice-over is perfect, the text is about Myzrael talking with the player. The [female titan's voice](https://www.wowhead.com/wotlk/npc=2755/myzrael#sounds) is more suitable in this case, instead of the narrator voice.

Progress: no playback.

Completion: Despite the voice-over is perfect, the text is about Myzrael talking with the player. The [female titan's voice](https://www.wowhead.com/wotlk/npc=2755/myzrael#sounds) is more suitable in this case, instead of the narrator voice.

**Comment by carevass** [comment]
This should be fixed with new audio in AI_VoiceOverData_VanillaExtra_v1.5.
